tests/PackageSelection/PackageSelectionTest.php

```diff
@@ -446,13 +471,13 @@ public function dataSelect(): array
                 $packages['alpha'],
                 $packages['epsilon'],
                 $packages['gamma1'],
+                $packages['gamma2'],
+                $packages['gamma3'],
```

Comment:
I updated this test. `project-eta` requires `"vendor/project-gamma": ">=1"`. It seems obvious that all versions of `gamma` are eligible.